README.md

```diff
@@ -81,24 +84,25 @@ export AWS_SECRET_ACCESS_KEY=...
 export AWS_ACCESS_KEY_ID=...
 ```
 
-If you need to specify your credentials manually, you can use a pre-configured instance of the [AWS SQS](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/SQS.html) client:
+If you need to specify your credentials manually, you can use a pre-configured instance of the [SQS Client](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-sqs/classes/sqsclient.html) client and chose from [a valid credentials provider](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/)
 
 ```js
-const { Consumer } = require('sqs-consumer');
-const AWS = require('aws-sdk');
+import { Consumer } from 'sqs-consumer';
+import { SQSClient } from '@aws-sdk/client-sqs';
+import { fromIni } from '@aws-sdk/credential-provider-ini';
 
-AWS.config.update({
-  region: 'eu-west-1',
-  accessKeyId: '...',
-  secretAccessKey: '...'
+const credentials = fromIni({
```

Comment:
I think this is quite complex for some people and not everyone sets up AWS SDKs in this way, would prefer using the old fixed values rather than the fromIni command, which complicates things a bit.

Users can work this out if they want to do it this way.